pkg/datastore/test/tuples.go

```diff
@@ -574,8 +574,7 @@ func CreateAlreadyExistingTest(t *testing.T, tester DatastoreTester) {
 	require.NoError(err)
 
 	_, err = common.WriteTuples(ctx, ds, core.RelationTupleUpdate_CREATE, tpl1)
-	require.Error(err)
-	require.Contains(err.Error(), "could not CREATE")
```

Comment:
I actually explicitly did not do it this way because we might change the error type coming out, but the *message* should always reference that the create could not occur; I'm fine leaving this change in now, but we should also check that the error message still contains at least part of what we expect

Reply:
well my goal was precisely following the principle of not using messages as interface. And the type is in `internal` anyway?

Reply:
Normally I agree, but since error messages are sent out to the API caller, I find it important to verify their contents too

Reply:
fair point. So we can haz both 😄